Resources/Prototypes/Entities/Objects/Tools/flare.yml

```diff
@@ -61,7 +61,7 @@
       - !type:FadeBehaviour # have the radius start small and get larger as it starts to burn
         id: turn_on
         interpolate: Linear
-        maxDuration: 45.0 #duration of light
+        maxDuration: 225.0 #duration of light
```

Comment:
I think this was probably intentionally less than the actual lifetime of the flare. I assume it was meant to get brighter over time, and then stay at the max brightness for 120 seconds.  If this was somehow causing flares to stop working early, there is something wrong with how `FadeBehaviour` or `LightBehaviour` works

Reply:
Actually the problem just seems to be that the point light component doesn't have `netsync: false`.
There's then a separate problem where the fade out and fade in animations have a noticeable jump. The fade-out should start with a radius of 10 and the energy flicker should be unchanged (i.e., still have a start/end of 6/15)

Reply:
Alright, I believe I understand what you mean, maxduration is the time it's supposed to actually fill out to it's full radius.

after adding netsync false it is able to actually to keep to the glow duration as listed above.